server/embed/etcd.go

```diff
@@ -162,6 +162,11 @@ func StartEtcd(inCfg *Config) (e *Etcd, err error) {
 
 	backendFreelistType := parseBackendFreelistType(cfg.BackendFreelistType)
 
+	warningUnaryRequestDuration, err := cfg.parseWarningUnaryRequestDuration()
+	if err != nil {
+		return e, err
+	}
```

Comment:
The comment https://github.com/etcd-io/etcd/pull/14414#discussion_r1020816764 is still valid.

Reply:
@ahrtr I was following this grouping of parsing functions https://github.com/etcd-io/etcd/blob/main/server/embed/etcd.go#L158
`parseCompactionRetention` and `parseBackendFreelistType`.
But since this change is about flags (not values), I've moved into config.parse per your suggestion.

Reply:
Points from my side:
1. The new method `parseWarningUnaryRequestDuration` looks good to me;
2. I don't think it's good to add too much configuration parsing code inside function `StartEtcd`, and the example you followed might need to be moved into `(cfg *config) `parse as well. But it's unrelated to this PR, so feel free to resolve it in a separate PR.